core/src/state.rs

```diff
@@ -93,15 +92,13 @@ impl State {
     }
 }
 
-/*
-TODO: write macro for DRY reducer functions
-macro_rules! reducer {
-    ($func_name:ident) => (
-        fn reducer(old_state: Rc<$state_type>, action: &_Action) -> Rc<$state_type>  {
-            // The `stringify!` macro converts an `ident` into a string.
-            println!("You called {:?}()",
-                     stringify!($func_name));
-        }
-    )
-}
-*/
+// TODO: write macro for DRY reducer functions
+// macro_rules! reducer {
+//     ($func_name:ident) => (
+//         fn reducer(old_state: Rc<$state_type>, action: &_Action) -> Rc<$state_type>  {
+//             // The `stringify!` macro converts an `ident` into a string.
+//             println!("You called {:?}()",
+//                      stringify!($func_name));
+//         }
+//     )
+// }
```

Comment:
Shouldn't we just delete this dead code and make a ticket out of that TODO?

Reply:
Good idea 👍